src/ui/properties/VideoNodeEditor.js

```diff
@@ -8,7 +8,7 @@ import StringInput from "../inputs/StringInput";
 import { VideoProjection } from "../../editor/objects/Video";
 import VideoInput from "../inputs/VideoInput";
 import { Video } from "styled-icons/fa-solid/Video";
-import AudioSourceProperties from "./AudioSourceProperties";
+import AudioParamsProperties from "./AudioParamsProperties";
```

Comment:
We discussed this a bit yesterday.. I wonder if `audio-source' is a better name for this than `audio-params`